File: bytetrackDemo_OC/bytetrackDemo_OC/ViewController.m
//
//  ViewController.m
//  bytetrackDemo_OC
//
//  Created by 孙良 on 2022/9/2.
//

#import "ViewController.h"
#import <Bytetrack/Bytetrack.h>
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    UIButton*openBtn=[[UIButton alloc]initWithFrame:CGRectMake([UIScreen mainScreen].bounds.size.width/2.0-50, 200, 100, 100)];
    [openBtn setTitle:@"打开信使" forState:UIControlStateNormal];
    [openBtn setTitleColor:[UIColor blueColor] forState:UIControlStateNormal];
    [openBtn addTarget:self action:@selector(openClick) forControlEvents:UIControlEventTouchUpInside];
    [self.view addSubview:openBtn];
    // Do any additional setup after loading the view.
}

//MARK: 打开信使
-(void)openClick{
    
    [Bytetrack presentMessenger];
}

@end

File: bytetrackDemo_Swift/bytetrackDemo_Swift/ViewController.swift
//
//  ViewController.swift
//  bytetrackDemo_Swift
//
//  Created by 孙良 on 2022/9/2.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let screenWidth = UIScreen.main.bounds.width
        let btn = UIButton(frame: CGRect(x: screenWidth/2.0-50, y: 200,  width: 100, height: 100))
        btn.setTitle("打开信使", for: .normal)
        btn.setTitleColor(UIColor.systemBlue, for: .normal)
        btn.addTarget(self, action: #selector(btnClicked), for: .touchUpInside)
        view.addSubview(btn)
        // Do any additional setup after loading the view.
    }

     @objc func btnClicked(){
        Bytetrack.presentMessenger()//打开信使
       }
}

